extensions/cornerstone-dicom-sr/src/utils/srInspection.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ConceptNameCodeSequence,
+  ConceptNameCodeSequenceItem,
+  ContentSequence,
+  ContentSequenceItem,
+  DICOMStandardReport,
+} from './srTypes';
+
+export function getCodeMeaningFromConceptNameCodeSequence(
+  conceptNameCodeSequence: ConceptNameCodeSequence
+): string {
+  let item: ConceptNameCodeSequenceItem = conceptNameCodeSequence[0];
+  const { CodeMeaning } = item;
+  return CodeMeaning ?? "";
```

Comment:
Why not 
{
   return conceptNameCodeSequence?.[0]?.CodeMeaning || ""
}

Really, all you are doing is extracting path values, and a single extractor is easier to manage.